packages/block-library/src/navigation/style.scss

```diff
@@ -60,10 +60,11 @@
 		// Sub-menus Flyout
 		& > li > ul {
 			background: #fff;
-			-webkit-box-shadow: -2px 4px 4px -1px rgba(97, 97, 97, 0.153);
-			-moz-box-shadow: -2px 4px 4px -1px rgba(97, 97, 97, 0.153);
-			border-radius: 4px;
-			box-shadow: -2px 4px 4px -1px rgba(97, 97, 97, 0.153);
+			-webkit-box-shadow: 0 3px 30px rgba(25, 30, 35, 0.1);
```

Comment:
Is there a reason you're not using straight black (`rgba( 0, 0, 0, x)`) or white ( `rgba( 255, 255, 255, x )` )?

Reply:
This is directly from existing shadows in Gutenberg so pulling from those.